### PR TITLE
I900 911 action milestone modals

### PR DESCRIPTION
**Pull request by Serhiy91**

### CODE REVIEW

File: app/imports/ui/react/components/EntityModal.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  formTarget,
+  isSubmitBtn,
+}) => (
+  <Modal {...{ isOpen, toggle }}>
+    <ModalHeader
+      renderLeftButton={() => isSubmitBtn ? <Button onClick={toggle}>Close</Button> : null}
+      renderRightButton={props => (
+        <SaveButton
+          isSaving={loading || props.loading}
+          onClick={!isSubmitBtn ? toggle : undefined}
```
Comment (Dartv) [inline app/imports/ui/react/components/EntityModal.js]: Why do you need the `isSubmitBtn` prop? Can't you just use the `toggle` prop?

File: app/imports/ui/react/goals/containers/ActionsMilestonesListContainer.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+import { lenses } from 'plio-util';
+import connectUI from 'redux-ui';
+import withStateToggle from '../../helpers/withStateToggle';
+import ActionsMilestonesList from '../components/ActionsMilestonesList';
+
+export default compose(
+  connectUI(),
+  withStateToggle(false, 'isOpen', 'toggle'),
+  withProps(({ goal }) => ({
+    items: reject(view(lenses.isCompleted), goal.points),
```
Comment (Dartv) [inline app/imports/ui/react/goals/containers/ActionsMilestonesListContainer.js]: `view(lenses.isCompleted)` is already available as `isCompleted` from `plio-util`

File: app/imports/ui/react/milestones/components/MilestoneSubcard.js [hunk 3]
```diff
@@ -19,6 +21,11 @@ const StyledHeader = styled.div`
   }
 `;
 
+const enhance = compose(
```
Comment (Dartv) [inline app/imports/ui/react/milestones/components/MilestoneSubcard.js]: See the comment for the action subcard

File: app/imports/ui/react/milestones/containers/NewMilestoneFormContainer.js [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+import { withProps } from 'recompose';
+
+import { Mutation, Fragment } from '../../../../client/graphql';
+import { namedCompose } from '../../helpers';
+import { updateGoalFragment } from '../../../../client/apollo';
+import NewMilestoneForm from '../components/NewMilestoneForm';
+
+const { GOAL_CARD } = Fragment;
+
+export default namedCompose('GoalMilestonesSubcardContainer')(
```
Comment (Dartv) [inline app/imports/ui/react/milestones/containers/NewMilestoneFormContainer.js]: name

File: app/imports/ui/react/milestones/containers/NewMilestoneFormContainer.js [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+    },
+  }) => ({
+    linkedTo: {
+      _id,
+      title,
+      sequentialId,
+    },
+    color,
+  })),
+  graphql(Mutation.CREATE_MILESTONE, {
```
Comment (Dartv) [inline app/imports/ui/react/milestones/containers/NewMilestoneFormContainer.js]: Same as actions

File: app/imports/ui/react/actions/containers/GeneralActionEditFormContainer.js [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+  graphql(LINK_DOC_TO_ACTION, { name: LINK_DOC_TO_ACTION.name }),
+  graphql(UNLINK_DOC_FROM_ACTION, { name: UNLINK_DOC_FROM_ACTION.name }),
+  withHandlers({
+    loadLinkedDocs: ({ organizationId, goals: actionGoals }) => () => client.query({
+      query: Query.GOAL_LIST,
+      variables: { organizationId },
+    }).then(({ data: { goals: { goals } } }) => ({
+      options: mapRejectedEntitiesToOptions(actionGoals, goals),
+    })),
+    onLink: ({ [LINK_DOC_TO_ACTION.name]: mutate, _id, goals }) =>
```
[the remark this answers is not in the document]
Comment (Serhiy91) [reply app/imports/ui/react/actions/containers/GeneralActionEditFormContainer.js]: No we do not have it. I have removed it from GoalActionsSubcardContainer

File: app/imports/ui/react/components/EntityModal.js (hunk 1 above)
Comment (Serhiy91) [reply app/imports/ui/react/components/EntityModal.js]: No I can not. I always have toggle handler.

**Comment by Serhiy91** [comment]
@Dartv fixed

**Comment by Dartv** [comment]
I often get this error:

**Comment by Serhiy91** [comment]
@Dartv fixed

**Comment by Serhiy91** [comment]
@Dartv fixed

**Pull request merged**